xsl/dita2html.xsl

```diff
@@ -26,16 +26,19 @@
     <xsl:text>layout: </xsl:text>
     <xsl:apply-templates select="." mode="jekyll-layout"/>
     <xsl:text>&#xA;</xsl:text>
-    <xsl:text>title: "</xsl:text>
+    <xsl:text>title: '</xsl:text>
     <xsl:apply-templates select="*[contains(@class, ' topic/title ')]" mode="text-only"/>
-    <xsl:text>"&#xA;</xsl:text>
-    <xsl:text>index: "</xsl:text>
+    <xsl:text>'&#xA;</xsl:text>
+    <xsl:text>description: '</xsl:text>
+    <xsl:apply-templates select="*[contains(@class, ' topic/shortdesc ')]" mode="text-only"/>
```

Comment:
Noticed this syntax only catches cases where `<shortdesc>` appears by itself, but not within `<abstract>`.

Not sure how to handle both.